drivers/syslog/Kconfig

```diff
@@ -407,5 +407,25 @@ config SYSLOG_REGISTER
 	---help---
 		This option will support register the syslog channel dynamically.
 
+config SYSLOG_STDOUT
+	bool "Redirect standard output stream of printf() and vprintf() to syslog"
+	default n
+	---help---
+		Redirect standard output stream of printf(), vprintf(), puts(), putchar() to syslog,
```

Comment:
standard output is not a per-function entity.
"standard output stream of printf()" doesn't make sense.